api/Metadata.json

```diff
@@ -38,7 +38,7 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": true,
+            "version_added": "1.0",
             "prefix": "webkit"
```

Comment:
This is missing from Opera, but in this case the Opera data is false, so wrong in a more easily detectable way. It's fine to leave that wrong for now.